### PR TITLE
service: bcat: Stub CreateDeliveryCacheStorageService

### PR DESCRIPTION
**Pull request by german77**
Simple stub for the service. Allows "groove coaster wai wai party" to boot.

### CODE REVIEW

**Comment by AbandonedCart** [comment]
This is what I got when testing the commit -
```
ld: error: undefined symbol: skyline::service::bcat::IDeliveryCacheStorageService::IDeliveryCacheStorageService(skyline::DeviceState const&, skyline::service::ServiceManager&)
>>> referenced by construct_at.h:38 (include/c++/v1/__memory/construct_at.h:38)
>>>               CMakeFiles/skyline.dir/src/main/cpp/skyline/services/bcat/IServiceCreator.cpp.o:(std::__1::shared_ptr<skyline::service::bcat::IDeliveryCacheStorageService> std::__1::allocate_shared<skyline::service::bcat::IDeliveryCacheStorageService, std::__1::allocator<skyline::service::bcat::IDeliveryCacheStorageService>, skyline::DeviceState const&, skyline::service::ServiceManager&, void>(std::__1::allocator<skyline::service::bcat::IDeliveryCacheStorageService> const&, skyline::DeviceState const&, skyline::service::ServiceManager&))
```
I'm double checking to make sure it isn't a local issue, but figured I would let you know.

**Comment by PixelyIon** [comment]
@german77 You'll need to manually add the file to `CMakeLists.txt`, we were planning to move to it being blobbed but haven't yet.

File: app/src/main/cpp/skyline/services/bcat/IDeliveryCacheStorageService.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "IDeliveryCacheStorageService.h"
+
+namespace skyline::service::bcat {
+    IDeliveryCacheStorageService::IDeliveryCacheStorageService(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager) {}
+
```
Comment (PixelyIon) [inline app/src/main/cpp/skyline/services/bcat/IDeliveryCacheStorageService.cpp:8]: Extra space.

File: app/src/main/cpp/skyline/services/bcat/IDeliveryCacheStorageService.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace skyline::service::bcat {
+    /**
+     * @brief IDeliveryCacheStorageService is used to create files instances for BCAT
+     * @url https://switchbrew.org/wiki/BCAT_services#IDeliveryCacheStorageService
+     */
+    class IDeliveryCacheStorageService : public BaseService {
+      public:
+        IDeliveryCacheStorageService(const DeviceState &state, ServiceManager &manager);
+
```
Comment (PixelyIon) [inline app/src/main/cpp/skyline/services/bcat/IDeliveryCacheStorageService.h:16]: Extra space.

**Review by PixelyIon — CHANGES_REQUESTED** [review]
Great job, this just has two minor nits regarding spacing and should be ready to merge after that.

**Pull request merged**